backdrop/transformers/dispatch.py

```diff
@@ -1,12 +1,19 @@
 import importlib
 import logging
 
+from os import getenv
+
+from backdrop.core.log_handler import get_log_file_handler
+
 from worker import app, config
 
 from performanceplatform.client import AdminAPI, DataSet
 
-
-logger = logging.getLogger(__name__)
+GOVUK_ENV = getenv("GOVUK_ENV", "development")
```

Comment:
Only to have parity with the other backdrop entry points which use the same name.